ref/feature/openidConnectServer/examples.adoc

```diff
@@ -8,7 +8,13 @@ Add the OP Open Liberty feature and any other needed features to the server.xml
 Define an OAuth service provider.
 Any user that is authorized to use OpenID Connect must also be mapped to the authenticated OAuth-role.
 
-In the following example, the OP expects the client’s SSL port to be set to 443:
+The following example assumes using basic registry which is available for purposes of development/test.
```

Comment:
Change 
"The following example assumes using basic registry which is available for purposes of development/test." 
to 
"The following example assumes that you are using a basic registry, which is available for development and test purposes."